sdk/core/azure-mgmt-core/azure/mgmt/core/polling/arm_polling.py

```diff
@@ -118,7 +130,9 @@ def set_initial_status(self, pipeline_response):
         # type: (PipelineResponseType) -> str
         """Process first response after initiating long running operation.
 
-        :param azure.core.pipeline.PipelineResponse response: initial REST call response.
+        :param azure.core.pipeline.PipelineResponse pipeline_response: initial REST call response.
```

Comment:
Why here you use `azure.core.pipeline.PipelineResponse` but `PipelineResponseType` in `get_final_get_url`?

Reply:
I missed that, going off of the # typehints here and making both pipelineResponseType